docs/alternator/compatibility.md

```diff
@@ -250,6 +250,11 @@ is different, or can be configured in Alternator:
   So for example, if you create a table whose name is 192 characters, you
   can't create a GSI whose name is longer than 29 characters.
 
+* DynamoDB's DescribeTable will return information about table (table's size
+  in bytes), that according to a documentation can lag behind up to 6 hours.
+  `alternator_describe_table_info_timeout_in_seconds` parameter allows
+  to change this timeout - default value in seconds is set to 21600 (6 hours).
```

Comment:
It may be useful to also document what this size measures in Alternator. It's certainly not obvious. It measures sstables - not the actual data, and it measures all 3 replicas, etc. I don't know how much users care about this detail, but I don't think we have it documented anywhere else?